editors/emacs/Portfile

```diff
@@ -101,19 +101,17 @@ if {$subport eq $name || $subport eq "emacs-app"} {
 }
 
 if {$subport eq "emacs-devel" || $subport eq "emacs-app-devel"} {
-    set date        2022-09-20
+    PortGroup       github 1.0
+
+    github.setup    emacs-mirror emacs 802671493d7c18569dac47ca91a4b7d6e693aff6
     epoch           5
-    version         [string map {- {}} ${date}]
-    revision        0
+    version         20221107
 
-    fetch.type      git
-    git.url         --shallow-since=${date}T00:00:00 https://github.com/emacs-mirror/emacs.git
-    git.branch      e5de29aa47e96f675b7f4ef3a7f84bd30d68ea0f
+    master_sites    ${github.master_sites}
```

Comment:
Thanks for moving to `PortGroup github`. It's less surprising this way.

Do we need to set `master_sites` here? I think the call to `github.setup` should set it, but maybe there's some magic that it does to not overwrite values set earlier?

Reply:
`github.setup` defines `default master_sites`. This Portfile is specify at begining `master_sites gnu` => I must switch to use PG's master_sites.

Reply:
OK thanks.